src/App.css

```diff
@@ -1,38 +0,0 @@
-.App {
```

Comment:
- [ ]While I appreciate the effort and time you have put into this but there is something you need to work on for your project to be completed,  the app is running well but I noticed that it is throwing an error, below point_down: 
![Screenshot from 2023-04-08 22-02-25](https://user-images.githubusercontent.com/88894525/230740489-a07e6818-0c22-4353-a91a-883e942aded7.png)

_note: In order to ensure that the app is functioning optimally, I recommend fixing this error as soon as possible. The error message indicates that the file "./App.css" could not be found in the specified directory, which is causing the application to fail to compile. To resolve this issue, please make sure that the file exists in the correct location and that the import statement in your "./App.jsx" file is correctly referencing the file._

Reply:
Well said @Alfredbis29 👍🏻